### PR TITLE
Bugfix when admin edit someone report become owner of this report

### PR DESCRIPTION
**Pull request by kbeker**
Resolves https://github.com/Code-Poets/sheetstorm/issues/350 https://github.com/Code-Poets/sheetstorm/issues/351

### CODE REVIEW

File: employees/views.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ class ReportDetailBase(UpdateView):
 
     def get_initial(self) -> dict:
         initial = super().get_initial()
-        initial.update({"author": self.request.user})
+        initial.update({"author": Report.objects.get(pk=self.kwargs["pk"]).author})
```
Comment (rwrzesien) [inline employees/views.py]: Report should be already available as `self.object`. If it is not, please get it with `self.get_object()`. This method takes into consideration all updates made ie. in `self.queryset` or `get_queryset` method.

File: employees/views.py [hunk 2]
```diff
@@ -262,7 +262,7 @@ class ReportDetailView(
...
```
Comment (rwrzesien) [inline employees/views.py]: Same here, please use `self.object` or `self.get_object()` (preferably first).

File: employees/views.py (hunk 1 above)
Comment (kbeker) [reply employees/views.py]: Done

File: employees/views.py (hunk 2 above)
Comment (kbeker) [reply employees/views.py]: Also done

**Pull request merged**